src/rules.rs

```diff
@@ -518,7 +538,7 @@ fn paren_on_top_level(element: &SyntaxElement) -> bool {
         NODE_SELECT | NODE_PAREN | NODE_APPLY => paren_on_top_level(&parent.into()),
         _ => false,
     }
-}
+}*/
```

Comment:
In emacs, it's generally faster to comment code out by selecting it and using <kbd>M-;</kbd> shortcut.

Reply:
I am aware of that but idk, it's bad habit I think.